ext/hal/altera/README

```diff
@@ -0,0 +1,11 @@
+The sources in this directory are imported from Altera
```

Comment:
Please follow the template provided in doc/contribute/code_component_README

Reply:
@nashif , i have actually provided info like Origin/Purpose/URL etc.. in the initial commit message itself. Also i have referred README's of other ext/hal/ folders are none of them are having this info README.
The README are simply having info like how they can be downloaded and what they do.

If you want, i can add that info in README as well.